docs/instrumenting-rails.md

```diff
@@ -1,6 +1,44 @@
 # Instrumenting Rails
 
-TODO
+Install the instrumentation library by adding it to your project's `Gemfile`:
+
+``` ruby
+gem "opentelemetry-instrumentation-rails", "~> 0.24"
+```
+
+or
+
+```shell
+bundle add opentelemetry-instrumentation-rails --version "~> 0.24"
+```
+
+Configure OpenTelemetry to use all available instrumentation libraries using the
```

Comment:
```suggestion
Configure OpenTelemetry to use all available instrumentation libraries by adding the
```

Reply:
For me one thing that's missing is where to put this code, and even OTel's readme doesn't explain that. One logical place would be `config/initializers`, but I'm familiar with all intricacies of hooking and I don't know if that will work for all cases - though I usually test it this way, and I've never had any issues.

Reply:
Closest possible place to what they have in the [example](https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/instrumentation/rails/example/trace_request_demonstration.ru), ie. before `Rails.application.initialize!`, would be `config/environment`, although it feels awkward.

Reply:
The Otel Ruby docs mention doing it in initializers, https://opentelemetry.io/docs/instrumentation/ruby/automatic/ -- I've updated this doc to also mention you'd put this in an initializer for Rails.